### PR TITLE
Applied the apply button.

### PR DESCRIPTION
**Pull request by ikedam**
Introduced the apply button to "Manage UpdateSites"

* When succeeds
apply_success
* When failed
apply_failure

### CODE REVIEW

**Comment by ikedam** [comment]
@lanwen
Would you review this change?

File: src/main/java/jp/ikedam/jenkins/plugins/updatesitesmanager/UpdateSitesManager.java [hunk 1]
```diff
@@ -161,7 +164,7 @@ public HttpResponse doUpdate(@Sites List<UpdateSite> managed) throws ServletExce
         Jenkins.getInstance().getUpdateCenter().getSites().replaceBy(newSitesList);
         Jenkins.getInstance().getUpdateCenter().save();
 
-        return HttpResponses.redirectViaContextPath("/manage");
+        FormApply.success(req.getContextPath() + "/manage").generateResponse(req, rsp, null);
```
Comment (ikedam) [inline src/main/java/jp/ikedam/jenkins/plugins/updatesitesmanager/UpdateSitesManager.java]: Reference implementation: https://github.com/jenkinsci/jenkins/blob/jenkins-1.609/core/src/main/java/hudson/security/GlobalSecurityConfiguration.java#L82

**Comment by lanwen** [comment]
LGTM :+1:

**Comment by ikedam** [comment]
Thanks for the review.

**Pull request merged**